parachains/install-polkadot-sdk.md

```diff
@@ -1,18 +1,25 @@
 ---
-title: Install Polkadot SDK Dependencies
-description: Install everything you need to begin working with Substrated-based blockchains and the Polkadot SDK, the framework for building blockchains.
+title: Install Polkadot SDK
+description: Install all required Polkadot SDK dependencies, set up the SDK itself, and verify that it runs correctly on your machine.
 categories: Basics, Tooling
 ---
 
-# Install Polkadot SDK Dependencies
+# Install Polkadot SDK
 
-This guide provides step-by-step instructions for installing the dependencies you need to work with the Polkadot SDK-based chains on macOS, Linux, and Windows. Follow the appropriate section for your operating system to ensure all necessary tools are installed and configured properly.
+This guide provides step-by-step instructions for installing the Polkadot SDK on macOS, Linux, and Windows. The installation process consists of two main parts:
+
+- **Installing dependencies**: Setting up Rust, required system packages, and development tools.
+- **Building the Polkadot SDK**: Cloning and compiling the Polkadot SDK repository.
+
+Follow the appropriate section for your operating system to ensure all necessary tools are installed and configured properly.
+
+## Install Dependencies
```

Comment:
Sorry for all the back and forth, I didn't notice this before, but this is just an empty ##. I think the following sections are supposed to be h3s and h4s, but I like to avoid h4s if possible.

So I'd remove this h2 and just update the other ones to be like ` ## Install Dependencies: macOS`